src/ResourceManagement/TrafficManager/TrafficManagerManagement/Properties/AssemblyInfo.cs

```diff
@@ -17,11 +17,11 @@
 using System.Resources;
 using System.Runtime.InteropServices;
 
-[assembly: AssemblyTitle("Microsoft Azure Dns Management Library")]
-[assembly: AssemblyDescription("Provides Microsoft Azure Dns management functions for managing the Microsoft Azure Dns service.")]
+[assembly: AssemblyTitle("Microsoft Azure Traffic Manager Management Library")]
+[assembly: AssemblyDescription("Provides Microsoft Azure Traffic Manager management functions for managing the Microsoft Azure Traffic Manager service.")]
 
 [assembly: AssemblyVersion("0.9.0.0")]
-[assembly: AssemblyFileVersion("0.9.1.0")]
+[assembly: AssemblyFileVersion("0.9.2.0")]
```

Comment:
I forgot that. The version that I originally meant to launch to public was 0.9.2